### PR TITLE
"execution expired" port from 3.2.1

### PR DESCRIPTION
**Pull request by xenolog**

### CODE REVIEW

File: deployment/puppet/neutron/manifests/agents/dhcp.pp [hunk 1]
```diff
@@ -20,6 +20,18 @@
...
          before  => Package['neutron-dhcp-agent'],
        }
+    } else {
+       file { '/etc/init/neutron-dhcp-agent.override':
+         replace => 'no',
+         ensure  => 'present',
+         content => 'manual',
+         mode    => 644,
+         before  => Package['neutron-dhcp-agent'],
+       }
```
Comment (xdel) [inline deployment/puppet/neutron/manifests/agents/dhcp.pp]: arrow is missing

File: deployment/puppet/neutron/manifests/agents/l3.pp [hunk 2]
```diff
@@ -9,18 +9,33 @@
...
+        before  => Package['neutron-l3'],
+      }
+    } else {
+      file { '/etc/init/neutron-l3-agent.override':
+        replace => 'no',
+        ensure  => 'present',
+        content => 'manual',
+        mode    => 644,
+        before  => Package['neutron-l3'],
+      }
```
Comment (xdel) [inline deployment/puppet/neutron/manifests/agents/l3.pp]: same

File: deployment/puppet/neutron/manifests/server.pp [hunk 3]
```diff
@@ -12,29 +12,35 @@
...
+    # * neutron-metadata-agent
+    # because we need STOP neutron-metadata-agent here
+    #
+    file { '/etc/init/neutron-metadata-agent.override':
+      replace => 'no',
+      ensure  => 'present',
+      content => 'manual',
+      mode    => 644,
+    } -> Package["$server_package"]
+    if $service_provider != 'pacemaker' {
```
Comment (xdel) [inline deployment/puppet/neutron/manifests/server.pp:36]: upstart then, for make condition more precise

**Comment by aglarendil** [comment]
Remove override ordering fixes, please. Let's fix this stuff separately. Andrey's request is better than yours.

**Comment by aglarendil** [comment]
rebase ex_expired fixes only, please

File: deployment/puppet/neutron/manifests/server.pp (hunk 3 above)
Comment (xenolog) [inline deployment/puppet/neutron/manifests/server.pp:36]: what do you mean?

**Comment by xenolog** [comment]
rebased to nightly master.

File: deployment/puppet/neutron/manifests/server.pp (hunk 3 above)
Comment (xdel) [inline deployment/puppet/neutron/manifests/server.pp:36]: nevermind, it was about pointing exact init system it such if
for such strict match you may want to specify upstart everywhere or keep previous case layout(it is all inside big Ubuntufied if, anyway)

File: deployment/puppet/neutron/manifests/agents/ovs.pp [hunk 4]
```diff
@@ -10,59 +10,54 @@
...
-    package { 'neutron-plugin-ovs-agent':
+    Package['neutron'] -> Package["$ovs_agent_package"]
+    package {"$ovs_agent_package":
       name   => $::neutron::params::ovs_agent_package,
     }
   } else {
     $ovs_agent_package = $::neutron::params::ovs_server_package
   }
 
+  if $::operatingsystem == 'Ubuntu' {
```
Comment (xdel) [inline deployment/puppet/neutron/manifests/agents/ovs.pp:31]: I`d suggest to rebase this to the current master.

File: deployment/puppet/neutron/manifests/server.pp (hunk 3 above)
Comment (xenolog) [inline deployment/puppet/neutron/manifests/server.pp:36]: I could not guess what you wanted to say. 
It it's anything important -- please make a patch.

**Comment by xdel** [comment]
+1

**Pull request merged**